cmd/mtr-exporter/main.go

```diff
@@ -6,13 +6,14 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/robfig/cron/v3"
 	"log"
 	"net/http"
 	"os"
-
-	"github.com/robfig/cron/v3"
 )
 
+var hosts arrayFlags
```

Comment:
you introduce a "global" variable while the existing code tries to minimize that. in general it is favourable to adjust to the existing style.